dotcom-rendering/src/components/Card/components/LI.tsx

```diff
@@ -64,10 +65,18 @@ const decideDivider = (
 	offsetBottomPaddingOnDivider: boolean,
 	paddingSize: string,
 	containerPalette?: DCRContainerPalette,
-) =>
-	offsetBottomPaddingOnDivider
-		? verticalDividerWithBottomOffset(paddingSize, containerPalette)
-		: verticalDivider(containerPalette);
+	verticalDividerColour?: string,
+) => {
+	const borderColour =
+		verticalDividerColour ??
+		(containerPalette &&
+			decideContainerOverrides(containerPalette).border.container) ??
+		neutral[86];
```

Comment:
If we’re importing this, should we use the non-deprecated version: `palette.neutral[86]`?